aztec/src/main/kotlin/org/wordpress/aztec/AztecTagHandler.kt

```diff
@@ -154,6 +170,29 @@ class AztecTagHandler(val context: Context, val plugins: List<IAztecPlugin> = Ar
         return false
     }
 
+    private fun handleCheckboxInput(output: Editable, context: Context, attributes: Attributes): Boolean {
```

Comment:
Here we convert from structure like: 
```
<ul>
<li><input type="checkbox" checked />Item</li>
</ul>
```

to something like

```
<ul>
<li checked="true>Item</li>
</ul>
```
This means that we remove the `input` element and we set the `checked` attribute on the `li` container. We also need to change the parent `ul` type to task list in order to draw the checkboxes.

Reply:
Good point, added in 42caab404109705e8ffc956fd58e5159027c57ca